pom.xml

```diff
@@ -818,7 +818,6 @@
           <version>4.6.1</version>
           <configuration>
             <sendJavaToScalac>false</sendJavaToScalac>
-            <scalaVersion>${scala2.12.version}</scalaVersion>
```

Comment:
what version will be used now that this is no longer specified?
was this always wrong or how is this related now to generating scala doc?

Reply:
setting this version globally resulted in a failure when the gatling module was executed. All of the other modules (except deltalake were already specifiying scala 2.12). Only the gatling module requires scala 2.13 but as I said, setting the version globally here resulted in issues